app/models/overlay/member.rb

```diff
@@ -16,7 +16,7 @@ class Member
 
       # There can be at most one of this types per visualization
       UNIQUE_TYPES = [
-          'header', 'search', 'layer_selector', 'share', 'zoom', 'logo', 'loader', 'fullscreen'
```

Comment:
Is this because there are now two 'header' types? One for title and one for description?